modules/3d-tiles/src/lib/parsers/helpers/parse-3d-implicit-tiles.ts

```diff
@@ -7,6 +7,7 @@ import {getS2CellIdFromToken, getS2ChildCellId, getS2TokenFromCellId} from '../.
 import type {S2VolumeInfo} from '../../utils/obb/s2-corners-to-obb';
 import {convertS2BoundingVolumetoOBB} from '../../utils/obb/s2-corners-to-obb';
 import Long from 'long';
+import {Tiles3DLoaderOptions} from '../../..';
```

Comment:
I don't like us to import from index files like this, let's specify file name.
- It is unclear and easily leads to circular deps. 
- Also if the import is a type please use `import type`. It reduces risk of circular import issues as types are more flexible.